app/views/confirm_email.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import uk.gov.hmrc.play.views.html.helpers.form
+
+@(email: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(title = Messages("confirm_email.title"), bodyClasses = None) {
+
+ <h1 class="heading-large">@Messages("confirm_email.heading")</h1>
+
+ <p class="lede">@email</p>
```

Comment:
Can this be in a h2 tag instead of a p tag please. It causes an accessibility warning